feature-libs/cart/wish-list/styles/_wish-list.scss

```diff
@@ -21,6 +21,10 @@
     }
   }
 
+  .cx-empty-wish-list {
+    @include type('2', 'normal');
```

Comment:
`type(2)` seems quite big in here, let's use the empty default `type()` instead and aligned center please.

![default type and centered](https://github.com/SAP/spartacus/assets/15113219/87fb0b6d-56c9-4a8d-a914-3b8aac410f4d)

Reply:
Done. I've omitted the 'type()' mixin since result would be the same as default paragraph.